cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -499,8 +513,20 @@ def create(self, catalog_item_class, *args, **kwargs):
         Returns:
             An instance of catalog_item_class
         """
+        provisioning_entry_point = kwargs.pop("provisioning_entry_point", None)
```

Comment:
Why are these fields only passed to the collection create method, and not included as catalog item attributes? These are fields that are part of a catalog item definition, and `cat_item.fill_dict` property could just include them so that they're filled by the default fill mechanisms.

Reply:
If I add these three fields to the ```fill_dict``` method. then my observations are like:
1. When we execute ```cat_item.fill_dict``` then we can see these fileds are filled with our values. But it is not able to execute ```self.parent_view.modal.cancel.click()```. So that **pop-up** window not get cancelled after filling these fields.